lib/cmd/fh3/admin/environments/create.js

```diff
@@ -1,44 +1,35 @@
-var log = require("../../../../utils/log");
-
 module.exports = {
   'desc' : 'Creates an environment.',
-  'examples' : [{ cmd : 'fhc admin environments create --id=<environment id> --label=<label> --targets=<mbaasTargetId1>,<mbaasTargetId2>',
+  'examples' : [{ cmd : 'fhc admin environments create --id=<environment id> --label=<label> --target=<mbaasTargetId>',
     desc : 'Creates an environment'}],
-  'demand' : ['id', 'label', 'targets'],
+  'demand' : ['id', 'label', 'target'],
   'alias' : {},
   'describe' : {
     'id' : 'Unique identifier for your environment',
     'label' : 'A label describing your environment',
-    'targets' : 'Comma separated list of MBaaS Target hostnames',
+    'target' : 'unique identifier for an existing MBaaS Target',
     'autoDeployOnCreate' : 'option used to setup automatic deployment to this environment when creating new project/app.',
     'autoDeployOnUpdate' : 'option used to setup automatic deployment to this environment when making source code changes within app.',
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this trailing comma. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingComma)